test/test_ao_sparsity.py

```diff
@@ -16,7 +16,7 @@
 from ao.sparsity.test_sparsifier import TestNearlyDiagonalSparsifier  # noqa: F401
 
 # Pruner
-from ao.sparsity.test_pruner import TestBasePruner  # noqa: F401
+from ao.sparsity.test_pruner import TestBaseStructuredPruner  # noqa: F401
```

Comment:
should we move `test/ao/sparsity` to `test/ao/pruning`, and rename the tests?  cc @z-a-f , ideally that would be a separate PR

Reply:
For consistency, I think it would make sense to rename the test folder.